### ISSUE TITLE
Enlarge images / videos on click

### ISSUE THREAD

**Issue by vogti** [polypheny/Polypheny-DB#308]
The data view in Polypheny-UI makes it possible to view images and videos. The multimedia objects in the table view and card view are displayed rather small. It would be great to integrate a lightbox plugin. Preferably, it should be possible to display the information of the other columns in a caption at the bottom, but this is not a must.

A great plugin to use is: https://www.npmjs.com/package/ngx-lightbox

Please make sure that the animations are fast so that it is possible to switch quickly between the images.

**Comment by JAgrit20** [comment]
Sir, if you can pinpoint the file/folder where I shall implement the changes?

**Comment by nilshansen94** [comment]
Hi @JAgrit20, the table view and card view can be found in the following directories:
`Polypheny-DB-UI/src/app/components/data-view/data-card`
`Polypheny-DB-UI/src/app/components/data-view/data-table`
Maybe you have to work with the `media` component as well:
`Polypheny-DB-UI/src/app/components/data-view/media`.
To test, you will have to create a table with an IMAGE and VIDEO column and add some data. In the UI, when you select a table in the data view you can switch between the table, carousel and card views. It would be good to integrate the lightbox plugin into the table and card views.

**Comment by Pawanupadhyay10** [comment]
HI,i will like to contribute on it, Can you assign this issue to me.

**Comment by vogti** [comment]
Hi @Pawanupadhyay10, @JAgrit20 is already working on this issue, but there are several other interesting issues you can work on.

**Comment by JAgrit20** [comment]
@vogti thanks, I am working on this,
 if you can see my timeline which I have submitted as a proposal once for GSOC, please?

**Comment by BrokenArrow1404** [comment]
Hi, I would like to is this problem still available in GSOC 2022?

**Comment by heitzli** [comment]
Hi @BrokenArrow1404 it would be great if you could handle that issue. I assigned it to you.

**Comment by BrokenArrow1404** [comment]
> Hi @BrokenArrow1404 it would be great if you could handle that issue. I assigned it to you.

got it and thanks

**Comment by BrokenArrow1404** [comment]
Sorry, I can't continue working on this project due to personal reasons.

**Comment by vogti** [comment]
Ok, if you have time again, please do not hesitate to reach out to us :)

**Comment by Nishant-dobaria** [comment]
hello sir, I want to know if this problem still open?

**Comment by vogti** [comment]
Yes, this issue is still open. Do you want to take care of it?

**Comment by Nishant-dobaria** [comment]
yes with pleasure, I am not familiar with angular but in order to solve that problem I first learn little bit about angular and then I approach to that problem. 
problem statement is : Enlarge images and videos on click like this right?
![ngx-box](https://user-images.githubusercontent.com/77713587/167304859-8be69539-85fe-49be-9658-a51d2ea64b69.jpg)

**Comment by vogti** [comment]
Yes, exactly. Great, I have assigned you the issue.

**Comment by Nishant-dobaria** [comment]
I am extremely sorry for not solving that issue till now because after 4
days my 6th sem exams are starting but after exams I definitely start
contributing to this project.

**Comment by msaini0r** [comment]
@Nishant-dobaria Hi, I would like to work on this issue. If it is available.

**Comment by AK4266** [comment]
Hello, i would like to work on this issue

**Comment by deepankar2704** [comment]
Hello sir I am currently worked on this type of problem. Can I get a chance to solve this issue?

**Comment by Sanchit3004** [comment]
Hello @vogti sir, I am 3rd year B.E.(IT) student. I would like to work on this issue for my GSOC'24. Is this issue still open? If yes, then please assign it to me.

**Comment by AbhijitRaut04** [comment]
Can I work on this issue? I am a complete beginner.

**Comment by TejasSaraf** [comment]
Hii @vogti, I would like to work on this issue can you assign it to me?

**Comment by SidraAnsari** [comment]
Hi! I want to work on this issue is it open?
